Assets/BossRoom/Scripts/Shared/OnSpawnBehaviourEnabler.cs

```diff
@@ -0,0 +1,69 @@
+using System;
+using System.Collections.Generic;
+using Unity.Netcode;
+using UnityEngine;
+
+namespace Unity.Multiplayer.Samples.Utilities
```

Comment:
This file will be moved to the Utilities package when the PR will leave the draft status. I kept it here to make testing and iterating easier since ParrelSync doesn't sync the packages folder automatically